fine_tuning/evaluate_finetuned.py

```
@@ -0,0 +1,235 @@
+
+        # 예측 실행
+        print("🤖 Fine-tuned 모델로 예측 중...")
+        predictions = []
+
+        for i, (_, row) in enumerate(df.iterrows(), start=1):
+            print(f"   [{i}/{len(df)}] 예측 중...", end='\r')
+
+            try:
+                pred = self.categorize_single(row['review_text'])
+                predictions.append(pred)
+            except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)